sci-biology/salmon/salmon-0.5.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2015 Gentoo Foundation
```

Comment:
`repomam` should have informed you about the year. I take it you are not using `repoman` then?